packages/sync/src/Queue.php

```diff
@@ -239,7 +352,14 @@ function checkout_with_memory_limit( $max_memory, $max_buffer_size = 500 ) {
 		return $buffer;
 	}
 
-	function checkin( $buffer ) {
+	/**
+	 * Checkin the queue.
```

Comment:
Suggested a typo, but I think this could be make clearer. I'm not sure what "check in the queue" means TBH.

```suggestion
	 * Check in the queue.
```